views/beneficial-owner-type.html

```diff
@@ -112,12 +112,18 @@ <h2 class="govuk-heading-l govuk-!-margin-0">What <span class="govuk-visually-hi
           {% include "includes/list/managing-officer.html" %}
 
           {% if (hasTrusts) %}
-            {% set nextPage = "trust-information" %}
+            {% set nextPage = pageParams.urlToTrust %}
           {% else %}
-            {% set nextPage = "check-your-answers" %}
+            {% set nextPage = pageParams.urlToCheckYourAnswers %}
           {% endif %}
 
-          <a href= "/register-an-overseas-entity/{{ nextPage }}" role="button" draggable="false" class="govuk-button govuk-button--secondary govuk-!-margin-top- 6" data-module="govuk-button" id="submit">No more <span class="govuk-visually-hidden"> {{ hiddenTextValue }} </span> to add</a>
+          <a href= "{{ nextPage }}"
```

Comment:
As best as I can understand it, this looks like an elegant improvement. However, can we please aim for consistency with the rest of the app.

Reply:
There is no functional improvement, more improve readability
Line is too long

Reply:
This is just better line wrapping. OK to keep @markpit ?